app/main/main.js

```diff
@@ -67,6 +69,9 @@ function createWindow(tabInfo) {
   });
 
   browser.webContents.once('did-finish-load', () => {
+    let browserDidFinishLoadTime = new Date();
```

Comment:
That's what I thought too, but I checked inside one of the components and it was defined